candidate-testing.js

```diff
@@ -1,58 +1,87 @@
-const input = require('readline-sync');
+const input = require('readline-sync')
 
-// TODO 2: modify your quiz app to ask 5 questions //
+let candidateName= "";
+let question = "Who was the first American woman in space? ";
+let correctAnswer = "Sally Ride";
+let candidateAnswer = "";
+let questions = ["Who was the first American woman in space? ",
+"True or false: 5 kilometer == 5000 meters? ",
```

Comment:
Style comment:  I would recommend using indents here (lines 8-12).  You can use as many indents and returns as you want in the code.  Spacing helps make the code more readable.